handbook/data-stores/data-stores.md

```diff
@@ -0,0 +1,89 @@
+# MongoDB
+
+MongoDB is used by the majority of SystemLink services as the primary datastore. MongoDB Community Edition is included in the default installation of SystemLink will launch when the SystemLink server boots.
+
+!!!note
+    By default SystemLink stores all data on databases and the file system hosted on the SystemLink application server. This is referred to as a *single node* deployment. To improve the reliability of data storage and scale to larger volumes of data, remote data stores are recommended for all production deployments. This is referred to as a *multi node* deployment.
```

Comment:
I've been wondering this for a while... does anyone actually use single-node deployment over a long period of time?  Since multi-node is the recommended approach, I wonder how much real estate we should give single-node deployment across all our help content.

Also, we could put this at the start of the "SystemLink with Single Node..." section instead, since there's nothing the user does with this info unless they're reading that part. And that'd still be a pretty prominent location.

I really like this differentiation between single-node and multi-node deployment. I'd like to update R2 docs to match this terminology.

Reply:
So, up till now I expect the single node deployment to be the most common. Not because its the best approach, but rather that (up until recently) it was all we could do. I honestly thing we should be pushing the single node deployment case as eval only. 

> Also, we could put this at the start of the "SystemLink with Single Node..." section instead, since there's nothing the user does with this info unless they're reading that part. And that'd still be a pretty prominent location.
I agree let's hammer that point home. Its clear with this feedback that the hierarchy of the document needs to be adjusted as well.

Reply:
This is definitely in the "things doctors tell you to do" space of recommended, but also a lot harder, so not everyone is going to actually follow the advice.